File: src/model/treino.py
class Treino:
    def __init__(self,
                 id_treino: int = None,
                 id_aluno: int = None,
                 id_instrutor: int = None,
                 tipo: str = None,
                 horario: str = None,
                 dias_semana: str = None,
                 frequencia: int = 0):
        self.set_id_treino(id_treino)
        self.set_id_aluno(id_aluno)
        self.set_id_instrutor(id_instrutor)
        self.set_tipo(tipo)
        self.set_horario(horario)
        self.set_dias_semana(dias_semana)
        self.set_frequencia(frequencia)

    # Setters
    def set_id_treino(self, id_treino: int):
        self.id_treino = id_treino

    def set_id_aluno(self, id_aluno: int):
        self.id_aluno = id_aluno

    def set_id_instrutor(self, id_instrutor: int):
        self.id_instrutor = id_instrutor

    def set_tipo(self, tipo: str):
        self.tipo = tipo

    def set_horario(self, horario: str):
        self.horario = horario

    def set_dias_semana(self, dias_semana: str):
        self.dias_semana = dias_semana

    def set_frequencia(self, frequencia: int):
        self.frequencia = frequencia

    # Getters
    def get_id_treino(self) -> int:
        return self.id_treino

    def get_id_aluno(self) -> int:
        return self.id_aluno

    def get_id_instrutor(self) -> int:
        return self.id_instrutor

    def get_tipo(self) -> str:
        return self.tipo

    def get_horario(self) -> str:
        return self.horario

    def get_dias_semana(self) -> str:
        return self.dias_semana

    def get_frequencia(self) -> int:
        return self.frequencia

    # Representação
    def to_string(self):
        return (f"Treino: {self.get_id_treino()} | Aluno: {self.get_id_aluno()} | "
                f"Instrutor: {self.get_id_instrutor()} | Tipo: {self.get_tipo()} | "
                f"Horário: {self.get_horario()} | Dias: {self.get_dias_semana()} | "
                f"Frequência: {self.get_frequencia()}")

File: src/model/Treino.java
public class Treino {
    private Integer idTreino;
    private Integer idAluno;
    private Integer idInstrutor;
    private String tipo;
    private String horario;
    private String diasSemana;
    private Integer frequencia;

    public Treino() {}

    public Treino(Integer idTreino, Integer idAluno, Integer idInstrutor, String tipo, String horario, String diasSemana, Integer frequencia) {
        this.idTreino = idTreino;
        this.idAluno = idAluno;
        this.idInstrutor = idInstrutor;
        this.tipo = tipo;
        this.horario = horario;
        this.diasSemana = diasSemana;
        this.frequencia = frequencia;
    }

    public Integer getIdTreino() { return idTreino; }
    public void setIdTreino(Integer idTreino) { this.idTreino = idTreino; }

    public Integer getIdAluno() { return idAluno; }
    public void setIdAluno(Integer idAluno) { this.idAluno = idAluno; }

    public Integer getIdInstrutor() { return idInstrutor; }
    public void setIdInstrutor(Integer idInstrutor) { this.idInstrutor = idInstrutor; }

    public String getTipo() { return tipo; }
    public void setTipo(String tipo) { this.tipo = tipo; }

    public String getHorario() { return horario; }
    public void setHorario(String horario) { this.horario = horario; }

    public String getDiasSemana() { return diasSemana; }
    public void setDiasSemana(String diasSemana) { this.diasSemana = diasSemana; }

    public Integer getFrequencia() { return frequencia; }
    public void setFrequencia(Integer frequencia) { this.frequencia = frequencia; }

    @Override
    public String toString() {
        return String.format("Treino: %s | Aluno: %s | Instrutor: %s | Tipo: %s | Horário: %s | Dias: %s | Frequência: %s",
                idTreino, idAluno, idInstrutor, tipo, horario, diasSemana, frequencia);
    }
}
